schemas/2019-09-16/Microsoft.HealthcareApis.json

```diff
@@ -63,6 +63,24 @@
           "type": "string",
           "description": "An etag associated with the resource, used for optimistic concurrency when editing it."
         },
+        "identity": {
+            "type": "object",
+            "description": "Setting indicating whether the service has a managed identity associated with it.",
+            "properties": {
+                "type":{
+                    "description" : "Type of identity being specified, currently SystemAssigned and None are allowed.",
+                    "type": "string",
+                    "enum": [
+                        "SystemAssigned",
+                        "None"
+                    ],
+                    "x-ms-enum": {
```

Comment:
x-ms-enum isn't a JSON schema property

Reply:
Updated per the comments to add a definition for the identity type